spec/Types.tex

```diff
@@ -66,6 +68,13 @@ \section{Primitive Types}
   `imag' primitive-type-parameter-part[OPT]
   `complex' primitive-type-parameter-part[OPT]
   `string'
+  `enum'
```

Comment:
`enumerated` ?

Reply:
It wasn't listed here before, and I don't want to add it now :)

Reply:
(That's mostly because I'm expecting we will deprecate it in favor of just using `enum`).